miso-web/src/main/resources/internal/miso.properties

```diff
@@ -69,6 +69,13 @@ miso.detailed.sample.enabled:false
 ## multiple scanners can be comma-separated
 # miso.visionmate.servers:scanner1:127.0.0.1:8000
 
+## config for links to external run report services.
+## multiple links can be comma-separated.
+## format: <PlatformType>:<link text>:<use HTTPS? (boolean)>:<link domain>:<link port>:<link page identifier>:<run field>
```

Comment:
Could we simplify this to `<Platform>|<link text>|<URI_with_placeholders>` where placeholders include `{id}`, `{name}`, and `{alias}`?

e.g. `Illumina|Acquacotta Run Report|http://rshiny.gsi.oicr.on.ca:3838/instances/acquacotta/?run={alias}`

Backslash (\) would be a preferable separator for multiple values too. All of `:,~` are allowed in URIs, but none of `|\{}` are

Reply:
Rats, I see I was looking at an older version of what characters are allowed in URIs. Will update the separator to `\` and suffer the regex consequences.